frontend/model/contracts/group.js

```diff
@@ -592,6 +592,22 @@ DefineContract({
         }
       }
     },
+    'gi.contracts/group/inviteRevoke': {
+      validate: (data, { state, getters, meta }) => {
+        objectOf({
+          inviteSecret: string // NOTE: simulate the OP_KEY_* stuff for now
+        })(data)
+
+        if (!state.invites[data.inviteSecret]) {
+          throw new TypeError(L('The link does not exist.'))
+        }
+      },
+      process ({ data, meta }, { state, getters }) {
+        console.debug('inviteRevoke:', data, state.invites)
```

Comment:
Extraneous debug comments not needed here. All of the existing SBP logging will automatically log this information anyway.